src/stars.css

```
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 12px;
+  width: 13px;
+  margin: 2px 5px 2px 2px;
+}
+
+.stars > :last-child {
+  margin-right: 0;
+}
+
+.stars--1 > :nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2 > :nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--3 > :nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--5 > :nth-child(-n + 5) {
+  background-image: url('images/star-active.svg');
+}
```

you can combine it :D 
```suggestion
.stars--1 > :nth-child(-n + 1), 
.stars--2 > :nth-child(-n + 2),
.stars--3 > :nth-child(-n + 3),
.stars--4 > :nth-child(-n + 4),
.stars--5 > :nth-child(-n + 5) {
  background-image: url('images/star-active.svg');
}
```

---

I didn't know this could be done like that, now it looks even smoother! 👍